x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/ilm/IndexLifecycleInitialisationTests.java

```diff
@@ -374,80 +374,7 @@ public void testMasterDedicatedDataDedicated() throws Exception {
         assertBusy(() -> {
             LifecycleExecutionState lifecycleState = LifecycleExecutionState.fromIndexMetadata(client().admin().cluster()
                 .prepareState().execute().actionGet().getState().getMetaData().index("test"));
-            assertThat(lifecycleState.getStep(), equalTo(TerminalPolicyStep.KEY.getName()));
-        });
-    }
-
-    public void testMasterFailover() throws Exception {
```

Comment:
was the deletion of this test intentional? if so shall we do it in a different commit so we have some semantics as to why it was removed?

Reply:
This was a test that didn't really do much outside of what we already have tested elsewhere, in addition, it was using a fake step that was near impossible to make work with the way that phases now operate, so I removed it.

I don't think a separate commit is that useful, since this ends up squashed into a single commit anyway?

Reply:
Ah, yeah, you're right (not sure how should we signal the reasoning behind these changes for future reference - commit message?)